### ISSUE TITLE
Problem about accuracy

### ISSUE THREAD

**Issue by ujscjj** [thuml/MDD#1]
Hello,thank you for your sharing.
I have a problem. When I run your codes, the eval accuracy is always 0. Just as bleow.
Cloud please tell me something about it?

init_lr: 0.004
lr_scheduler: {'gamma': 0.001, 'decay_rate': 0.75, 'type': 'inv'}
optim: {'type': 'sgd', 'params': {'momentum': 0.9, 'nesterov': True, 'weight_decay':                                                0.0005, 'lr': 0.004}}
start train...
Train iter:   0%|▍                                                                          | 500/100000 [03:21<11:58:55,  2.31it/s]
 {'accuracy': tensor(0, device='cuda:0')}        | 4/16 [00:01<00:05,  2.14it/s]
Train iter:   1%|▋                                                                         | 1000/100000 [06:48<10:18:59,  2.67it/s]
 {'accuracy': tensor(0, device='cuda:0')}        | 8/16 [00:03<00:03,  2.60it/s]
Train iter:   2%|█▏                                                                         | 1500/100000 [10:14<9:52:52,  2.77it/s]
 {'accuracy': tensor(0, device='cuda:0')}▊      | 12/16 [00:04<00:01,  2.74it/s]
Train iter:   2%|█▌                                                                         | 2000/100000 [13:31<8:34:56,  3.17it/s]
 {'accuracy': tensor(0, device='cuda:0')}                | 0/16 [00:00<?, ?it/s]
Train iter:   2%|█▊                                                                        | 2500/100000 [16:58<12:21:09,  2.19it/s]
 {'accuracy': tensor(0, device='cuda:0')}        | 4/16 [00:01<00:06,  1.86it/s]
Train iter:   3%|██▎                                                                        | 3000/100000 [20:25<9:56:52,  2.71it/s]
 {'accuracy': tensor(0, device='cuda:0')}        | 8/16 [00:03<00:03,  2.64it/s]
Train iter:   4%|██▋                                                                        | 3500/100000 [23:53<9:39:06,  2.78it/s]

**Comment by hlqwerty** [comment]
Hi, this problem may be caused by using a mismatched pytorch version. When I run the code with pytorch 0.4 or 1.0, this problem also happened. We will release code for recent pytorch versions in the future.

**Comment by hlqwerty** [comment]
Hi, this problem may be caused by using a mismatched pytorch version. When I run the code with pytorch 0.4 or 1.0, this problem also happened. We will release code for recent pytorch versions in the future.
